src/EdFi.AnalyticsMiddleTier.DataStandard31/Asmt/MSSQL/0014-View-StudentAssessmentFact-Create.sql

```diff
@@ -0,0 +1,126 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+IF EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.VIEWS WHERE TABLE_SCHEMA = 'analytics' AND TABLE_NAME = 'asmt_StudentAssessmentFact')
+BEGIN
+	DROP VIEW analytics.asmt_StudentAssessmentFact
+END
+GO
+
+CREATE VIEW analytics.asmt_StudentAssessmentFact AS
+
+	SELECT
+        CONCAT(StudentAssessment.AssessmentIdentifier, '-', StudentAssessment.Namespace, '-', StudentAssessment.StudentAssessmentIdentifier, '-', StudentAssessment.StudentUSI) AS StudentAssessmentKey,
+		CASE WHEN SASOA.StudentUSI IS NULL 
+                THEN NULL 
```

Comment:
Replace Null with ''

Reply:
Add a test case to validate the empty StudentObjectiveAssessmentKey and ObjectiveAssessmentKey.